packages/workspace/src/workspace/state/in_mem.ts

```diff
@@ -71,6 +74,55 @@ export const buildInMemState = (
     await (await stateData()).elements.delete(id)
   }
 
+  const updateAccounts = async (accounts?: string[]): Promise<void> => {
+    const data = await stateData()
+    const newAccounts = accounts ?? await awu(getUpdateDate(data).keys()).toArray()
+    return getUpdateDate(data).setAll(
+      awu(newAccounts.map(s => ({ key: s, value: new Date(Date.now()) })))
+    )
+  }
+
+  const updateStatePathIndex = async (
+    unmergedElements: Element[],
+    removedElementsFullNames?: Set<string>,
```

Comment:
does not need to be optional (AFAICT, only one place calls this, and it passes this argument)
```suggestion
    removedElementsFullNames: Set<string>,
```